pkg/activator/handler/context.go

```diff
@@ -54,3 +54,11 @@ func RevisionFrom(ctx context.Context) *v1.Revision {
 func RevIDFrom(ctx context.Context) types.NamespacedName {
 	return ctx.Value(revCtxKey{}).(*revCtx).revID
 }
+
+func RevAnnotations(ctx context.Context, annotation string) string {
```

Comment:
Nit, name implies multiple annotations, but we are filtering for one. Maybe just `RevAnnotationValue`?

Reply:
Ok.